packages/app-store/templates/audit-log-implementation/api/ping.ts

```diff
@@ -0,0 +1,47 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import z from "zod";
+
+import type { AuditLogEvent } from "@calcom/features/audit-logs/types";
+import { defaultResponder } from "@calcom/lib/server";
+import prisma from "@calcom/prisma";
+
+import AuditLogManager from "../lib/AuditLogManager";
+import { appKeysSchema } from "../zod";
+
+const pingEvent: AuditLogEvent = {
+  action: "ping.connection",
+  actor: {
+    id: "jackson@boxyhq.com",
+    name: "Jackson",
+  },
+  target: {
+    name: "tasks",
+    type: "Tasks",
+  },
+};
+
+const ZPingInputSchema = z.object({
+  credentialId: z.number(),
+});
+
+export async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
This api endpoint is used by the SystemStatus component. Thought it would be nice to have some sort of visual feedback. If credentials and system is working then users will know right away. Also pinging incurs in an event being reported. This is one of the events that can not/should not be disabled at all.